testsuite/python3.py

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env python3
+from typing import ClassVar, List
```

Comment:
So we support several versions of Python 3 including those without this module. I wonder if it wouldn't hurt to have a separate `python36.py` file that uses the typing module and annotated variables, etc.

Thoughts?